flink-connectors/flink-connector-elasticsearch6/pom.xml

```diff
@@ -172,6 +172,14 @@ under the License.
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>org.apache.flink</groupId>
```

Comment:
Can we remove dependencies to `flink-table-api-java-bridge` or `flink-table-planner` already now? I think the elasticsearch module should be able to work with `flink-table-common` classes only, right?

Reply:
It still depends on `flink-table-api-java-bridge` and `flink-table-planner` as `Elasticsearch6UpsertTableSinkFactory` implements `StreamTableSinkFactory`.